README.md

```diff
@@ -1 +1,10 @@
-# Hello-Microverse
\ No newline at end of file
+## Hello-Microverse
+
+### This is my first project at Microverse. In this project will cover the following issues,
+
+Working with different branches.
+Using linters in code base.
+Add .gitignore file.
+Document a repo in a professional way.
+Write a descriptive and understandable README file.
+Submit project for reviews.
```

Comment:
- It is very important you write a well-documented readme in order to make your repo look more professional. I'd suggest you use this [readme](https://github.com/microverseinc/readme-template/blob/master/README.md) template from Microverse 😄

Reply:
After a long time, I solved this problem according to the requirements. Please review the solutions.